src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.component.ts

```diff
@@ -162,6 +178,9 @@ export class FeedbackAnalysisComponent {
         if (filters.testCases && filters.testCases.length > 0) {
             count += filters.testCases.length;
         }
+        if (filters.errorCategories && filters.errorCategories.length > 0) {
```

Comment:
```suggestion
        if (filters.errorCategories?.length) {
```
Also true for the lines above ( if (filters.testCases && filters.testCases.length > 0) {`)